conda_smithy/linter/hints.py

```diff
@@ -250,12 +251,13 @@ def hint_noarch_python_use_python_min(
             ("run", "python >={{ python_min }}", run_reqs),
             ("test.requires", "python {{ python_min }}", test_reqs),
         ]:
-            test_syntax = syntax.replace("{{ python_min }}", "9999")
-
             if recipe_version == 1:
                 syntax = syntax.replace(
                     "{{ python_min }}", "${{ python_min }}"
                 )
+                test_syntax = syntax
+            else:
+                test_syntax = syntax.replace("{{ python_min }}", "9999")
```

Comment:
@beckermr, is `9999` intentional? Looks like it could be remnants from a test?

Reply:
It is intentional.